src/NServiceBus.Persistence.CosmosDB.LogicalOutbox.AcceptanceTests/ConfigureEndpointCosmosDBPersistence.cs

```diff
@@ -22,7 +22,7 @@ public Task Configure(string endpointName, EndpointConfiguration configuration,
         persistence.CosmosClient(SetupFixture.CosmosDbClient);
         persistence.DatabaseName(SetupFixture.DatabaseName);
 
-        persistence.Container(SetupFixture.ContainerName, SetupFixture.PartitionPathKey);
+        //persistence.Container(new ContainerInformation(SetupFixture.ContainerName, new PartitionKeyPath(SetupFixture.PartitionPathKey)));
```

Comment:
Technically we could introduce another set of permutations for the outbox if we want. I left this for now commented out because I tested both permutations during the refactoring by enabling this and removing the behavior set.